scripts/db_migrator.py

```diff
@@ -520,8 +510,6 @@ def version_2_0_0(self):
         Version 2_0_0.
         """
         log.log_info('Handling version_2_0_0')
-        self.migrate_config_db_port_table_for_auto_neg()
-        self.set_version('version_2_0_1')
         return 'version_2_0_1'
 
     def version_2_0_1(self):
```

Comment:
We can remove this section and keep 2_0_0 as latest

Reply:
fixed